pkg/build/builder/docker.go

```diff
@@ -57,6 +57,9 @@ func NewDockerBuilder(dockerClient DockerClient, buildsClient client.BuildInterf
 
 // Build executes a Docker build
 func (d *DockerBuilder) Build() error {
+	if d.build.Spec.Source.Git == nil && d.build.Spec.Source.Binary == nil && d.build.Spec.Source.Dockerfile == nil && d.build.Spec.Source.Images == nil {
+		return fmt.Errorf("Must provide a value for at least one of source, binary, images, or dockerfile")
+	}
```

Comment:
On Wed, Apr 6, 2016 at 10:26 AM, Michal Fojtik <notifications@github.com>
wrote:

> In pkg/build/builder/docker.go
> <https://github.com/openshift/origin/pull/8219#discussion_r58714097>:
>
> > @@ -57,6 +57,9 @@ func NewDockerBuilder(dockerClient DockerClient, buildsClient client.BuildInterf
> >
> >  // Build executes a Docker build
> >  func (d *DockerBuilder) Build() error {
> > +	if d.build.Spec.Source.Git == nil && d.build.Spec.Source.Binary == nil && d.build.Spec.Source.Dockerfile == nil && d.build.Spec.Source.Images == nil {
> > +		return fmt.Errorf("Must provide a value for at least one of source, binary, images, or dockerfile")
> > +	}
>
> but lowercase the error message :-)
>

Just pushed the update - thanks

> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8219/files/ee4a4a380229ec883ecd10a3f128fb63218cc618#r58714097>
>